app/controllers/RiderController.java

```diff
@@ -19,14 +27,101 @@ public RiderController(RiderRepository riderRepository) {
         this.riderRepository = riderRepository;
     }
 
-    public Result index() {
-        return ok("RIDER REPO");
+    public CompletionStage<Result> getRiders() {
+        return riderRepository.getAllRiders().thenApplyAsync(riders -> {
+            return ok(riders);
+        }).exceptionally(ex -> {
+            Result res = null;
+            switch (ExceptionUtils.getRootCause(ex).getClass().getSimpleName()){
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this "switch" statement by "if" statements to increase readability. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1301)